gn3/errors.py

```diff
@@ -106,6 +106,20 @@ def handle_generic(exc: Exception) -> Response:
     return resp
 
 
+def handle_llm_error(exc: Exception) -> Response:
+    """ Handle llm erros if not handled  anywhere else. """
+    resp = jsonify({
```

Comment:
can we also log this somewhere? My suggestion would be: https://github.com/jnduli/genenetwork3/blob/d4b00115b0b970b16af56f63b06cb20827695659/gn3/errors.py#L82